### PR TITLE
[Parser] Fix using without config

### PR DESCRIPTION
**Pull request by a-lyupa**
Don't abort if there is no config

Signed-off-by: Anastasia Lyupa <a.lyupa@samsung.com>

### CODE REVIEW

File: combirepo/config_parser.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def initialize_config(config_file_path):
     global default_path
```
Comment (ilya-palachev) [inline combirepo/config_parser.py:45]: BTW, above link (help_url = "http://confluence.rnd.samsung.ru/display/TC/Config+file+format") is not visible from outside. But this is not an issue of this patch.

File: combirepo/commandline_parser.py [hunk 2]
```diff
@@ -312,7 +312,12 @@ def __set_program_run_parameters(self, arguments):
         else:
             logging.basicConfig(level=logging_level)
 
-        config_parser.initialize_config(arguments.config)
+        if len(arguments.triplets) == 0:
+            gen_init_config = True
+        else:
+            gen_init_config = False
+
+        config_parser.initialize_config(arguments.config, gen_init_config)
```
Comment (ilya-palachev) [inline combirepo/commandline_parser.py:320]: For clarification: So we are generating a config if the user provides neither triplets through command line, nor configuration file, right?

**Review by ilya-palachev — COMMENTED** [review]
Seems that commit message need to be done more clear, for example, "Don't abort if there is neither config nor triplets in command line options".

File: combirepo/config_parser.py (hunk 1 above)
Comment (rayslava) [reply combirepo/config_parser.py:45]: Anyway we should remove link from source.
Maybe it'll be better to move this doc into additional man page?

Comment (ilya-palachev) [reply combirepo/config_parser.py:45]: ...or into wiki of this project

Comment (ilya-palachev) [reply combirepo/config_parser.py:45]: up to your decision

File: combirepo/commandline_parser.py (hunk 2 above)
Comment (a-lyupa) [reply combirepo/commandline_parser.py:320]: Yes, right.

Comment (ilya-palachev) [reply combirepo/commandline_parser.py:320]: OK, thanks! Then I'm OK to merge this pull request as-is. Maybe we just need to improve commit message a bit (see above comment).

Comment (a-lyupa) [reply combirepo/commandline_parser.py:320]: Done

**Review by ilya-palachev — APPROVED** [review]
Thanks! Looks good to me.

**Review by rayslava — APPROVED** [review]

**Pull request merged**